music_assistant/common/helpers/datetime.py

```diff
@@ -27,7 +27,7 @@ def now_timestamp() -> float:
     return now().timestamp()
 
 
-def future_timestamp(**kwargs) -> float:
+def future_timestamp(**kwargs: float) -> float:
```

Comment:
this feels a bit weird to type the kwargs this way - what are the actual (kw)args on the timedelta method ?

Reply:
Agree. It makes me not want to use kwargs anywhere tbh. But they need a type. I checked before i did this and timedelta has 7 args, and they are all floats.

I could add all 7 args as fully fleshed out kwargs?

Reply:
that is sad, is there no generic way to type kwargs ?
I guess dict[str, Any] but that is maybe a tiny bit too wide?

Reply:
I did find https://stackoverflow.com/a/63550734. There's an Unpack that you can use with kwargs to refer to a TypedDict. I can't say I like it that much. Would rather just type out the kwargs I think.